bee-network/src/service/controller.rs

```diff
@@ -0,0 +1,53 @@
+// Copyright 2020-2021 IOTA Stiftung
+// SPDX-License-Identifier: Apache-2.0
+
+use super::{
+    command::{Command, CommandSender},
+    event::{Event, EventReceiver},
+};
+
+#[derive(Debug, thiserror::Error)]
+pub enum Error {
+    #[error("Error sending command.")]
```

Comment:
even if this `Error` type is not exposed?

Reply:
`NetworkCommandSender`/`NetworkEventReceiver` are both exposed, and they deal with this type, so it would be useful for people who encounter these errors through their use, I think.

Reply:
done